release-team/release-team-removal.md

```diff
@@ -0,0 +1,48 @@
+# Resignation
+Voluntary resignation is outlined in the [shadow handbook](https://github.com/leonardpahlke/sig-release/blob/master/release-team/shadows.md). Role lead should discuss with Release Lead and Emiritus Advisor to find a replacement if they would like to step down.
+
+# Removal Process
+Remaining on the team without contributing creates an unhealthy environment for the rest of the team, and negatively affect the outcome of the release. We strive to be inclusive and empathetic to individuals circumstances. 
```

Comment:
```suggestion
Remaining on the team without contributing creates an unhealthy environment for the rest of the team, and negatively affects the outcome of the release. We strive to be inclusive and empathetic to individuals circumstances.
```

Reply:
At the end of the cycle, the leads of each team could confirm a list of the shadows abandon their duties and be removal from the github release team (https://github.com/kubernetes/sig-release/blob/master/releases/release-1.xx/release-team.md)